hms-app/src/components/HackathonSelectDropdown.tsx

```diff
@@ -27,38 +31,63 @@ export default function HackathonSelectDropdown(setHackathonID: Props) {
     )
   }
 
-  const hackathonMap = hackathonList.map((hackathon, index) => ({
-    value: hackathon.id,
-    label:
-      hackathon.title +
-      ' ' +
-      new Date(hackathon.startDate).toLocaleDateString() +
-      '-' +
-      new Date(hackathon.endDate).toLocaleDateString(),
-  }))
+  const futureAndPresentHackathons = hackathonList.filter((hackathon) => {
+    return new Date(hackathon.endDate) > today
```

Comment:
you probable want >= today and maybe even want to force today to be the start of the day